src/content/reference/react-dom/createPortal.md

```diff
@@ -29,13 +29,17 @@ To create a portal, call `createPortal`, passing some JSX, and the DOM node wher
 ```js
 import { createPortal } from 'react-dom';
 
+const portalNode = document.createElement('div')
+portalNode.id = 'portal'
+document.appendChild(portalNode)
```

Comment:
It's a little noisy to repeat this over and over throughout the doc. Maybe it would be better to declare this once up front and refer to something like `document.getElementById('portal')`?